mods/d2k/rules/structures.yaml

```diff
@@ -649,6 +649,9 @@ wall:
 	Sellable:
 		SellSounds: CHUNG.WAV
 	Guardable:
+	Explodes:
+		Weapon: WallExplode
+		EmptyWeapon: WallExplode
 	ThrowsShrapnel:
```

Comment:
Did some more testing and they spawned *exactly* 2 debris, 1 small and 1 large.
I noticed that our 'Pieces' represent the number for the respective list entry rather than min/max total, so I changed it to `1, 1`. Other buildings will need that fixed as well I guess, but not in this PR.